app/src/main/res/layout/widget_button.xml

```diff
@@ -10,44 +10,51 @@
     android:background="@drawable/widget_button_background"
     android:padding="4dp">
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/widgetImageButtonLayout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:paddingVertical="4dp"
+        android:layout_height="wrap_content"
```

Comment:
The size change will create visual problems with longer labels and/or smaller widget sizes:
![Widget with text overlapping icon](https://github.com/user-attachments/assets/19bb34c1-8915-48b0-88b0-a9e4b4c25b45)
(default size, and this label isn't even that long)

Reply:
This has not been resolved. It may be less obvious now, but text is cut off (note the bottom of the g/p), and especially for users with larger font sizes configured this will happen quite easily.
![image](https://github.com/user-attachments/assets/4d696cae-372b-4955-bf36-7ee8d63a01dd)

Reply:
Yeah I figured the issue is now caused by the text's size rather than its layout overlapping the above icon. I can reproduce it only if I manually set my font size in the phone's setting to the maximum, otherwise it looks good to me. Is that the case here as well?

Reply:
No, this is on the default text and display size for me (Pixel 7a; text size 2/7, display size 2/5).

Reply:
Please see [this review comment](https://github.com/home-assistant/android/pull/4594#pullrequestreview-2266977219).